lookout/style/typos_checker/README.md

```diff
@@ -26,7 +26,7 @@ $ go get github.com/src-d/lookout/...
 $ docker-compose up bblfsh postgres
 ```
 
-and then initialize **lookout** database
+Next, then initialize **lookout** database
```

Comment:
Next, initialize lookout database 
or Then, initialize lookout database

Reply:
thanks, I will fix it